index.css

```diff
@@ -0,0 +1,4 @@
+p {
+  color: red;
+  font-size: 3rem;
+}
```

Comment:
As mentioned in the previous comment, kindly use the correct `CSS selector` for the header text *"Hello Microverse!"*.